### PR TITLE
Phemex setMarginMode

### PR DESCRIPTION
**Pull request by Dan-krm**
Added setMarginMode to Phemex:
Cross is selected by setting leverage to 0, Isolated is 1x to 100x
```
node examples/js/cli phemex setMarginMode cross BTC/USD:BTC

phemex.setMarginMode (cross, BTC/USD:BTC)
2022-05-10T04:40:39.504Z iteration 0 passed in 552 ms

{ code: '0', msg: '', data: 'OK' }
```

### CODE REVIEW

File: js/phemex.js [hunk 1]
```diff
@@ -2932,6 +2933,33 @@ module.exports = class phemex extends Exchange {
...
+            throw new BadRequest (this.id + ' setMarginMode() marginType argument should be isolated or cross');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (market['type'] !== 'swap') {
+            throw new BadSymbol (this.id + ' setMarginMode() supports swap contracts only');
+        }
+        let leverage = this.safeInteger (params, 'leverage');
+        if (marginType === 'cross') {
+            leverage = 0;
```
Comment (kroitor) [inline js/phemex.js:2951]: Is there a reason for making the default value leverage equal to zero on cross margin?

Comment (Dan-krm) [reply js/phemex.js:2951]: The only way to set the margin mode to cross is by setting leverage to 0 for this exchange

**Comment by CloudTrauma** [comment]
Is this for python too?

**Comment by Dan-krm** [comment]
> Is this for python too?

Yes it should work for python as well

**Comment by kroitor** [comment]
@CloudTrauma yes, assuming you upgrade to the most recent version of CCXT.

**Comment by CloudTrauma** [comment]
Ok thnx

**Pull request merged**